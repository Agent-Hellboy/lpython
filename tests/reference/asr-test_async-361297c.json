{
    "basename": "asr-test_async-361297c",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_async.py",
    "infile_hash": "f4d737246effd50f1798a81f07042ad15a045e275448fe0226334f03",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_async-361297c.stderr",
    "stderr_hash": "3046c004d7c132ad57d4a01c3659bebfdbc291a592f26049efc0e397",
    "returncode": 1
}