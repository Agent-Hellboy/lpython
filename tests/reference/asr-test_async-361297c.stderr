Internal Compiler Error: Unhandled exception
Traceback (most recent call last):
[2m  File "[0m[1m[35m$DIR/src/bin/lpython.cpp[39m[0m[2m", line 1778[0m
    if (show_ast) {
[2m  File "[0m[1m[35m$DIR/src/bin/lpython.cpp[39m[0m[2m", line 233[0m
    std::cout << LCompilers::pickle(*asr, compiler_options.use_colors,
[2m  File "[0m[1m[35m$DIR/src/lpython/semantics/python_ast_to_asr.cpp[39m[0m[2m", line 7912[0m
    throw SemanticError("sizeof only accepts one argument, found " +
[2m  File "[0m[1m[35m$DIR/src/lpython/semantics/python_ast_to_asr.cpp[39m[0m[2m", line 4589[0m
    }
[2m  File "[0m[1m[35m$DIR/src/lpython/semantics/python_ast_to_asr.cpp[39m[0m[2m", line 3911[0m
    ASR::symbol_t *current_module_sym;
[2m  File "[0m[1m[35m$DIR/src/lpython/python_ast.h[39m[0m[2m", line 1883[0m
    void visit_stmt(const stmt_t &b) { visit_stmt_t(b, self()); }
[2m  File "[0m[1m[35m$DIR/src/lpython/python_ast.h[39m[0m[2m", line 1751[0m
    case stmtType::FunctionDef: { v.visit_FunctionDef((const FunctionDef_t &)x); return; }
[2m  File "[0m[1m[35m$DIR/src/lpython/python_ast.h[39m[0m[2m", line 1885[0m
    void visit_AsyncFunctionDef(const AsyncFunctionDef_t & /* x */) { throw LCompilersException("visit_AsyncFunctionDef() not implemented"); }
LCompilersException: visit_AsyncFunctionDef() not implemented
